requirements-dev.txt

```diff
@@ -7,6 +7,6 @@ pytest-asyncio
 pytest-snapshot
 coverage
 pytest-cov
+pytest-playwright
 pre-commit
 locust
-azure-storage-file-datalake==12.12.0
```

Comment:
@mattmsft This requirement is also in scripts/requirements.txt, so it doesn't need to be here as well, right?